binsrc/vsp/admin/admin_dav/bridge.vspx

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0"?>
-<?xml version="1.0"?>
```

Comment:
Why it was deleted ?

Reply:
There were originally two such lines.  There should be one.  (Accidental duplication of #750 within #760 may be reducing that to zero.)

Reply:
Ok